lib/modules/versioning/ubuntu/distribution.ts

```diff
@@ -0,0 +1,39 @@
+import dataFiles from '../../../data-files.generated';
+
+export type UbuntuDistroInfo = Record<string, string>;
+
+// Data file generated with:
+// ubuntu-distro-json-generate.mjs
+const ubuntuJsonKey = 'data/ubuntu-distro-info.json';
+
+const ubuntuDistroInfo: UbuntuDistroInfo = JSON.parse(
+  dataFiles?.get(ubuntuJsonKey) ?? ''
```

Comment:
🤔 do we really need those fallbacks? Error should be caught on ci before publish

Reply:
null/type-check related